### DISCUSSION TITLE
Bunch home automation

### DISCUSSION THREAD

**Discussion by ed-tobias** [ttscoff/bunch-docs#335] [Ideas]
For those that use Home Assistant for their home automation, an automation can be created and triggered by a webhook. The Home Assistant webhook API allows local webhook triggers using HTTP POST, PUT, HEAD, and GET requests. It can also be submitted from a shell cmd using curl. Bunch should be able to take advantage of this to set lighting, turn on fans or music etc.
see [Home Assistant webhooks](https://www.home-assistant.io/docs/automation/trigger/#webhook-trigger)

**Comment by ttscoff** [comment]
That’s a great tip. Bunch only supports GET right now, but I’ve considered adding POST capabilities. Which of course you can accomplish with a curl shell line right now.

Thanks,
Brett
